src/Translatable/Translatable.php

```diff
@@ -674,9 +674,9 @@ public function scopeOrWhereTranslationLike(Builder $query, $key, $value, $local
     /**
      * @return array
      */
-    public function toArray()
+    public function attributesToArray()
     {
-        $attributes = parent::toArray();
+        $attributes = parent::attributesToArray();
 
         if ($this->relationLoaded('translations') || $this->toArrayAlwaysLoadsTranslations()) {
```

Comment:
`if(!$this->relationLoaded('translations') && !$this->toArrayAlwaysLoadsTranslations())` removes the useless `else` here.